src/utils/gauges.ts

```diff
@@ -222,8 +226,8 @@ export async function getGaugeData(
             gaugeRelativeWeight: gaugeRelativeWeight,
             gaugeTotalSupply: gaugeTotalSupplies[index],
             workingSupply: gaugeWorkingSupplies[index],
-            workingBalances: gaugeWorkingBalances[index],
-            gaugeBalance: gaugeBalances[index],
+            workingBalances: gaugeWorkingBalances?.[index],
```

Comment:
Prefer to fall back to `Zero` instead of undefined for these values

Reply:
Updated!